README.md

```diff
@@ -1,12 +1,27 @@
 # Global Assessment of Natural Hazards towards Vulnerable and Unconnected Population (glanhavup)
```

Comment:
I think the readme looks good, but for now could you just remove the example results please? The example results graphics in the readme need more work, so I wouldn't put them public right now. I think we are fine just having the method for now.